tracekit.js

```diff
@@ -233,11 +233,10 @@ TraceKit.report = (function reportModuleWrapper() {
      */
     function processLastException() {
         var _lastExceptionStack = lastExceptionStack,
-            _lastArgs = lastArgs;
-        lastArgs = null;
+            _lastException = lastException;
         lastExceptionStack = null;
         lastException = null;
-        notifyHandlers.apply(null, [_lastExceptionStack, false].concat(_lastArgs));
+        notifyHandlers(_lastExceptionStack, false, _lastException);
```

Comment:
Okay, sounds good, guess we have a history of this change, so we'll know soon enough what this broke.